packages/messaging/src/util/constants.ts

```diff
@@ -15,81 +15,17 @@
  * limitations under the License.
  */
 
-export const DEFAULT_PUBLIC_VAPID_KEY = new Uint8Array([
-  0x04,
-  0x33,
-  0x94,
-  0xf7,
-  0xdf,
-  0xa1,
-  0xeb,
-  0xb1,
-  0xdc,
-  0x03,
-  0xa2,
-  0x5e,
-  0x15,
-  0x71,
-  0xdb,
-  0x48,
-  0xd3,
-  0x2e,
-  0xed,
-  0xed,
-  0xb2,
-  0x34,
-  0xdb,
-  0xb7,
-  0x47,
-  0x3a,
-  0x0c,
-  0x8f,
-  0xc4,
-  0xcc,
-  0xe1,
-  0x6f,
-  0x3c,
-  0x8c,
-  0x84,
-  0xdf,
-  0xab,
-  0xb6,
-  0x66,
-  0x3e,
-  0xf2,
-  0x0c,
-  0xd4,
-  0x8b,
-  0xfe,
-  0xe3,
-  0xf9,
-  0x76,
-  0x2f,
-  0x14,
-  0x1c,
-  0x63,
-  0x08,
-  0x6a,
-  0x6f,
-  0x2d,
-  0xb1,
-  0x1a,
-  0x95,
-  0xb0,
-  0xce,
-  0x37,
-  0xc0,
-  0x9c,
-  0x6e
-]);
+export const DEFAULT_SW_PATH = '/firebase-messaging-sw.js';
+export const DEFAULT_SW_SCOPE = '/firebase-cloud-messaging-push-scope';
 
-export const SUBSCRIPTION_DETAILS = {
-  userVisibleOnly: true,
-  applicationServerKey: DEFAULT_PUBLIC_VAPID_KEY
-};
+export const DEFAULT_VAPID_KEY =
+  'BDOU99-h67HcA6JeFXHbSNMu7e2yNNu3RzoMj8TM4W88jITfq7ZmPvIM1Iv-4_l2LxQcYwhqby2xGpWwzjfAnG4';
 
 export const ENDPOINT = 'https://fcmregistrations.googleapis.com/v1';
 
+/** Key of FCM Payload in Notification's data field. */
+export const FCM_MSG = 'FCM_MSG';
+
 export const FN_CAMPAIGN_ID = 'google.c.a.c_id';
```

Comment:
Yeah... while we're here should these be CONSOLE or COMPOSER rather than FN?

Reply:
Changed to `CONSOLE`.